src/use-masonry.tsx

```diff
@@ -196,6 +191,21 @@ export function useMasonry<Item>({
     estimateHeight(itemCount, itemHeightEstimate)
   );
 
+  // sort children to preserve the dom index
+  // that would help to preserve the tabbing order if it's required
+  const children = itemsToRender
+    .sort((a, b) => a.index - b.index)
```

Comment:
I'm concerned about this being the default behavior. I personally do not see the utility of this and I'm only entertaining it because others seem to want it. I'd be much more happy if this were an option a la `sortByIndex: boolean;`

Reply:
I've been thinking about it as well, adding yet another option would extend the api surface and can reduce simplicity. 
I can think of another way of getting the same behaviour by extending positioner in usercode, but I need to test it first.
Out of curiosity, are you concerned that it's a performance issue, potentially breaking change or something else?

Reply:
Yeah, I mean it feels icky to use binary search, red/black interval trees to achieve `O(log n)` and `O(log n + k)` complexity only to throw `O(n log n)` on top if it
![image](https://user-images.githubusercontent.com/86995/161467738-925be584-eb44-4d14-9588-a3d60cb28c47.png)